tests/Api/Response/Parser/FiltersParserTest.php

```diff
@@ -219,47 +197,8 @@ public function parseDataProvider()
                                 </item>
                             </items>
                         </filter>
-                        <filter>
-                            <name>price-text</name>
-                            <display>Preis</display>
-                            <select>single</select>
-                            <type>text</type>
-                            <items>
-                                <item>
-                                </item>
-                            </items>
-                        </filter>
-                        <filter>
-                            <name>color</name>
-                            <display>Farbe</display>
-                            <select>multiselect</select>
-                            <selectedItems>0</selectedItems>
-                            <type>color</type>
-                            <items>
-                                <item>
-                                    <name>lila</name>
-                                    <weight>0.068965516984463</weight>
-                                    <color>#BA55D3</color>
-                                </item>
-                                <item>
-                                    <name>rot</name>
-                                    <weight>0.068965516984463</weight>
-                                    <color>#FF0000</color>
-                                </item>
-                                <item>
-                                    <name>schwarz</name>
-                                    <weight>0.068965516984463</weight>
-                                    <color>#000000</color>
-                                </item>
-                                <item>
-                                    <name>weiß</name>
-                                    <weight>0.068965516984463</weight>
-                                    <color>#FFFFFF</color>
-                                </item>
-                            </items>
-                        </filter>
-                    </filters>
-                </searchResult>', [
+                        </main>
+                   ', [
```

Comment:
Put this on the same line as `</main>` (as it was before with `</searchResult>`).